rfc/2019-12-11-Talent-Plan-Courses.md

```diff
@@ -0,0 +1,31 @@
+# Proposal: Talent Plan Courses
+
+- Author(s): [Ling Jin](https://github.com/JinLingChristoher)
+- Last updated: 2019-12-11
+- Discussion at: [Incubating Program: Talent Plan Courses](https://github.com/pingcap/community/issues/130)
+
+There are plenty of people interested in TiDB but lacking background knowledge, so we propose to create a series of courses to help those people enter the field infrastructure systems and TiDB ecosystem. Course participants should have developed good knowledge and skills in system programming after finishing course materials.
+
+## Course design
+
+![course map](./course-ex.png)
+
+The above is the design of the course system. This series of courses include but not limited to the principles of computer system design, the programming language to be used, open-source cooperation, and more. The development of a distributed database system course is in progress, and more courses will be created in the future.
+
+## Rationale
+
+There are many open study resources online, our courses can borrow ideas from world-class universities and from MOOCs as well. The courses will be practice-concentrate, with well-designed programming assignments as the core. Reading materials including slides, notes is also irreplaceable.
```

Comment:
> Reading materials include slides, notes is also irreplaceable.

Can you explain this? I don't get it.

Reply:
The slide might be the most widely used form to represent teach contents, but the note is also important and should not be ignored.